.github/ISSUE_TEMPLATE/format_request.yml

```
@@ -0,0 +1,56 @@
+name: Request Format Support
+description: Request support for a new format.
+title: "[New Format]: "
+labels: ["enhancement", "data interfaces"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Thank you for your suggestion!
+
+        We welcome any ideas about how to make **NeuroConv** better for the community.
+
+        Please keep in mind that adding support for new formats may take some time.
+  - type: textarea
+    id: what-format
+    attributes:
+      label: What format would you like to see added to NeuroConv?
+      description: What acquisition hardware or software is responsible for producing the output files?
+    validations:
+      required: true
+  - type: textarea
+    id: format-documentation
+    attributes:
+      label: Does the format have any documentation?
+      description: A link or other details to rich documentation describing the folder/file structure.
```

```suggestion
      description: A link or other details to rich documentation describing the folder/file structure.
    - type: textarea
      id: format-api
      description: Do you know of any open source libraries for reading this data format?
```

---

```suggestion
      description: A link or other details to rich documentation describing the folder/file structure.
    - type: textarea
      id: format-api
      attributes:
        label: Existing APIs for format
        description: Do you know of any open source libraries for reading this data format?
```

I'd drop the NEO/SI bit. Without attaching links to those projects they may never have heard of them. Also the backend stuff is something we care about but not necessarily something a simple requester would need to know or check. Once we know the format name and suffixes we can also easily check it ourselves